src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -56,139 +58,140 @@ const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
     handleModalToggle();
   }
 
+  const hasCreatedManifest = typeof createManifestResponseData !== 'undefined';
+
+  const formHasError = errorCreatingManifest || hasSatelliteVersionsError;
+
+  const satelliteTypeOptions = satelliteVersions?.map((satelliteVersion: SatelliteVersion) => {
+    return (
+      <FormSelectOption
+        key={satelliteVersion.value}
+        value={satelliteVersion.value}
+        label={satelliteVersion.description}
+        validated={typeValidated}
+      />
+    );
+  });
+  const isValidManifestName = (manifestName: string) => {
+    if (
+      /^[0-9A-Za-z_.-]*$/.test(manifestName) &&
+      manifestName.length > 0 &&
+      manifestName.length < 99
+    ) {
```

Comment:
> For clarity I would make this logic a function that returns a bool called something like `isValidManifestName` so that when we glance at this if statement it is immediately clear what it is checking

With this comment I meant just a function that encapsulated the complex if - handling the rest in the useState I think makes the most sense.

The goal was just that it is clear what that if statement is checking